io/src/main/scala/sbt/io/IO.scala

```diff
@@ -85,7 +85,7 @@ object IO {
    * Note that Java standard library classes typically do not have a location associated with them.
```

Comment:
I would change the word location to File since they do have a location, it just isn't representable as a java.io.File.

Reply:
I wonder if that's true for all things module path, not just standard library on JDK 11.